lib/flappi/builder_factory.rb

```diff
@@ -76,12 +76,14 @@ def self.build_and_respond(controller)
       end
 
       # Merge in default values where one is defined and we don't have an actual parameter
```

Comment:
I'm not quite sure where I could hook in a test for those default params. Do we have some tests regarding that `build_and_respond` method already somewhere @richdrich?